radio/src/lua_api.cpp

```diff
@@ -144,31 +150,57 @@ static void luaPushLatLon(TelemetrySensor & telemetrySensor, TelemetryItem & tel
   lon = gpsLon / 1000000.0;
   if (telemetryItem.gps.longitudeEW == 'W') lon = -lon;
 
-  lua_createtable(L, 0, 2); /* push the result table */
-  lua_pushstring(L, "Lat");
-  lua_pushnumber(L, lat);
-  lua_settable(L, -3);
-  lua_pushstring(L, "Lon");
-  lua_pushnumber(L, lon);
-  lua_settable(L, -3);
+  lua_newtable(L);
+  lua_pushtablenumber(L, "lat", lat);
+  lua_pushtablenumber(L, "lon", lon);
+}
 
+static void luaPushTelemetryDateTime(TelemetrySensor & telemetrySensor, TelemetryItem & telemetryItem)
+{
+  luaPushDateTime(L, telemetryItem.datetime.year + 2000, telemetryItem.datetime.month, telemetryItem.datetime.day,
+                  telemetryItem.datetime.hour, telemetryItem.datetime.min, telemetryItem.datetime.sec);
+}
+
+static void luaPushCells(TelemetrySensor & telemetrySensor, TelemetryItem & telemetryItem)
+{
+  if (telemetryItem.cells.count == 0)
+    lua_pushinteger(L, (int)0); // returns zero if no cells
+  else {
+    lua_createtable(L, 0 /*telemetryItem.cells.count*/, 0);
```

Comment:
The change was made after my initial testing when I noticed that the convention for this module seemed to me using the lua_newtable() macro (which expands to lua_createtable(L, 0, 0)).

There are 13 references to lua_newtable() in this file.  Should they all be changed to explicit allocations?

David